torch/nested/_internal/nested_tensor.py

```diff
@@ -176,7 +210,7 @@ def _get_max_seqlen(self):
             # compute & cache
             max_val = _get_sdpa_extreme_seqlen(
                 torch.max,
-                self._offsets.diff() if self._lengths is None else self._lengths,
+                self.offsets().diff() if self._lengths is None else self._lengths,  #
```

Comment:
hm why is this necessary?

Reply:
mypy  :(
offsets() is guaranteed to return a Tensor based on its annotation. _offsets may return None